src/styles/mixins.less

```diff
@@ -362,12 +362,23 @@
   opacity: 0.9;
 }
 
+@keyframes fade-in-animation {
+  0% {
+      opacity: 0;
+  }
+  100% {
+      opacity: 1;
+  }
+}
+
 .m-video-play-overlay {
   .m-mask;
+  opacity: 0;
```

Comment:
Oh it's here already 😄

Reply:
True, I removed it in the `html5-player.less` file. Thanks!